packages/flutter/lib/src/cupertino/icon_theme_data.dart

```diff
@@ -27,18 +27,6 @@ class CupertinoIconThemeData extends IconThemeData with Diagnosticable {
     return resolvedColor == color ? this : copyWith(color: resolvedColor);
   }
 
-  /// Creates a copy of this icon theme but with the given fields replaced with
-  /// the new values.
-  @override
-  CupertinoIconThemeData copyWith({ Color? color, double? opacity, double? size, List<Shadow>? shadows }) {
```

Comment:
Why are we removing `copyWith`? Don't we still need it?

Reply:
Initially thought it was an unnecessary override, but it does return a more specific type, so added it back.